governance.md

```diff
@@ -0,0 +1,137 @@
+# Governance
+
+[mdn-browser-compat-data](https://github.com/mdn/browser-compat-data) (also often referred to as "BCD") is an open source project that depends on contributions from the community. Anyone may contribute to the project at any time by submitting code, participating in discussions, making suggestions, or any other contribution they see fit. This document describes how various types of contributors work within the mdn-browser-compat-data project.
+
+## Roles and Responsibilities
+
+### Everyone
+Everyone who is involved in any form with the project must abide by the project’s [Contribution Guidelines](https://github.com/mdn/browser-compat-data/blob/master/CODE_OF_CONDUCT.md). Everyone is expected to be respectful of every community member and to work collaboratively in the spirit of inclusion.
+
+### Users
+Users are community members who have a need for the project. They are typically consumers of the compat data. Anyone can be a User; there are no special requirements and the data is licensed under [CC0](https://github.com/mdn/browser-compat-data/blob/master/LICENSE). Common User contributions include evangelizing the project (e.g., display a link on a website and raise awareness through word-of-mouth), informing developers of strengths and weaknesses from a new user perspective, or providing moral support (a “thank you” goes a long way).
+
+Users who continue to engage with the project and its community will often become more and more involved. Such Users may find themselves becoming [Contributors](#Contributors), as described in the next section.
+
+### Contributors
```

Comment:
Peers and Owners have an easy to follow bulleted list of what they can do. Why not represent the things Contributors can do in this format too, to keep things consistent?

Reply:
Good idea! Done.